src/pages/MyJobs/ClientJobs/ClientJobs.tsx

```diff
@@ -7,19 +9,22 @@ import PageContainer, {
   NameText,
   Section,
   TitleText,
+  StyledPagination,
 } from "pages/MyJobs/ClientJobs/ClientJobsStyles";
 import { useTranslation } from "react-i18next";
+import { useGetJobPostQuery } from "services/jobPosts/setJobPostsAPI";
 import { SEARCH_WORK_ROUTE } from "utils/routeConsts";
 
-// Testing without backend connected
-const jobPosts: any[] | undefined = [
-  { id: "1", title: "Freelancing website", description: "Description" },
-  { id: "2", title: "Landing page design", description: "Description" },
-];
-
 function ClientJobs() {
   const { email } = useAuth();
   const { t } = useTranslation();
+  const [currentPage, setCurrentPage] = useState(1);
+  const [postsPerPage] = useState(3);
```

Comment:
as long as you don't provide setState this property is unchangeable. So why do you need to use useState, maybe it better made a const value?

Reply:
Had added this one because in the future we could use it in a filter or maybe in a "see more posts" feature. 

But to use it as a proper state I've now included a dropdown for the user to select how many posts per page he wants to see
![image](https://user-images.githubusercontent.com/61670871/187577507-9c2aa7a3-9dd0-40a8-8902-c30af7c0547a.png)

Also, I've taken the opportunity to change the fallback component to show a spinner instead of a text saying "Loading"